--- make.inc.example	2016-06-19 00:15:11.000000000 +0200
+++ make.inc.64	2016-09-17 19:29:06.054225293 +0200
@@ -16,11 +16,11 @@
 #  and handle these quantities appropriately. As a consequence, one 
 #  should not compile LAPACK with flags such as -ffpe-trap=overflow.
 #
-FORTRAN  = gfortran 
+FORTRAN  = x86_64-w64-mingw32-gfortran 
 OPTS     = -O2 -frecursive
 DRVOPTS  = $(OPTS)
 NOOPT    = -O0 -frecursive
-LOADER   = gfortran
+LOADER   = x86_64-w64-mingw32-gfortran
 LOADOPTS =
 #
 #  Comment out the following line to include deprecated routines to the
@@ -51,15 +51,15 @@
 #
 # CC is the C compiler, normally invoked with options CFLAGS.
 #
-CC = gcc
+CC = x86_64-w64-mingw32-gcc
 CFLAGS = -O3
 #
 #  The archiver and the flag(s) to use when building archive (library)
 #  If you system has no ranlib, set RANLIB = echo.
 #
-ARCH     = ar
+ARCH     = x86_64-w64-mingw32-ar
 ARCHFLAGS= cr
-RANLIB   = ranlib
+RANLIB   = x86_64-w64-mingw32-ranlib
 #
 #  Location of the extended-precision BLAS (XBLAS) Fortran library
 #  used for building and testing extended-precision routines.  The
